shell/pages/about.vue

```diff
@@ -108,6 +108,8 @@ export default {
         :to="{ name: 'diagnostic' }"
         class="btn role-primary"
         data-testid="about__diagnostics_button"
+        role="link"
```

Comment:
I think this should support space as it is a button

Reply:
changed the role to `button`, as it's a button on it's essence and added the `space` handler